feconf.py

```diff
@@ -500,9 +500,15 @@ def get_empty_ratings():
 # category.
 MINIMUM_SCORE_REQUIRED_TO_REVIEW = 10
 
+# The number of easy question skill difficulty.
+EASY_SKILL_DIFFICULTY = 0.3
+
```

Comment:
Drop this newline; also drop the newine between middle and hard. Put all these constants together.

Reply:
Done.